src/components/table-component/table.jsx

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+
+/* 
+ @author [Kurtney J.](https://github.com/xhundo)
+*/
+
+// Table component build with named spaced format
+
+const StyledCell = styled.div`
+  width: 343px;
+  height: 84px;
+  background: rgba(246, 246, 246, 0.1);
+`;
+
+const StyledHeading = styled.p`
+  font-size: 10px;
+  line-height: 12px;
+  color: rgba(246, 246, 246, 0.7);
```

Comment:
I realize figma might be slightly different, but can we use either `var(--bs-white75)` or `var(--bs-white60)` here, whichever is closer?

Reply:
I'm made those changes.